scrapers/PlumperPass.yml

```diff
@@ -7,19 +7,15 @@ sceneByURL:
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: 
-        selector: //div[@class="clearfix"]/h1/span/text()
-        postProcess:
-          - replace:
-              - regex: \"
-                with:
-      Details: //div[@class="movie-desc"]/text()
+      Title:
+        selector: //h2[@class="vidtitle"]
+      Details: //div[@class="vidinfo col-sm-12 col-md-12 col-lg-12 col-xl-3"]/p
```

Comment:
wouldn't `//div[h2[@class="vidtitle"]]/p` work ? and do all the scenes have a single p (just making sure, i havent spotted one with more)